z390test/src/main/groovy/org/z390/test/z390Test.groovy

```diff
@@ -0,0 +1,267 @@
+        /**
+         * Assemble
+         */
+        this.clean(asmFilename)
+        int rc = this.callZ390(asmFilename, 'mz390', args)
+        this.getOutput(asmFilename)
+        return rc
+    }
+
+    def lz390(Map kwargs=[:], String asmFilename, String... args) {
+        /**
+         * lz390
+         * !! NOTE !! Does not clean intermediate files
+         */
+        int rc = this.callZ390(asmFilename, 'lz390', args)
+        this.getOutput(asmFilename)
+        return rc
+    }
+
+    def asml(Map kwargs=[:], String asmFilename, String... args) {
+        /**
+         * Assemble and link
+         */
+        this.clean(asmFilename)
+        int rc = this.callZ390(asmFilename, 'mz390', args)
+        if (rc == 0) {
+            rc = this.callZ390(asmFilename, 'lz390', args)
+        }
+        this.getOutput(asmFilename)
+        return rc
+    }
+
+    def ez390(Map kwargs=[:], String asmFilename, String... args) {
+        /**
+         * ez390
+         * !! NOTE !! Does not clean intermediate files
+         */
+        int rc = this.callZ390(asmFilename, 'ez390', args)
+        this.getOutput(asmFilename)
+        return rc
+    }
+
+    def asmlg(Map kwargs=[:], String asmFilename, String... args) {
+        /**
+         * Assemble, link and go
+         */
+        this.clean(asmFilename)
+        int rc = this.callZ390(asmFilename, 'mz390', args)
+        if (rc == 0) {
+            rc = this.callZ390(asmFilename, 'lz390', args)
+            if (rc == 0) {
+                rc = this.callZ390(asmFilename, 'ez390', args)
+            }
+        }
+        this.getOutput(asmFilename)
+        return rc
+    }
+
+    def cblc(Map kwargs=[:], String cobFilename, String... args) {
+        /**
+         * cobol compile
+         */
+        this.clean(cobFilename)
+        int rc
+        rc = this.callZ390(cobFilename, 'zc390', *args)
+        if (rc != 0) return rc  // exit if issue
+
+        var cobfiledir = new File(cobFilename).getParent()
+        var cobOptions = ['BAL', 'NOLISTCALL', 'MAXGBL(1500000)',
+                          "SYSMAC(${basePath('zcobol', 'mac')}+${basePath('mac')})".toString(),
+                          "SYSCPY(${cobfiledir}+${basePath('zcobol', 'cpy')})".toString()
+        ]
+        rc = this.callZ390(cobFilename, 'mz390', *(cobOptions + args.toList()))
+        return rc
+    }
+
+    def cblclg(Map kwargs=[:], String cobFilename, String... args) {
+        /**
+         * cobol compile link and go
+         */
+        this.clean(cobFilename)
+        int rc
+        rc = this.callZ390(cobFilename, 'zc390', *args)
+        if (rc != 0) return rc  // exit if issue
+
+        var cobfiledir = new File(cobFilename).getParent()
+        var cobOptions = ['BAL', 'NOLISTCALL', 'MAXGBL(1500000)',
+                      "SYSMAC(${basePath('zcobol', 'mac')}+${basePath('mac')})".toString(),
+                      "SYSCPY(${cobfiledir}+${basePath('zcobol', 'cpy')})".toString()
```

Comment:
While it may be okay, you've deviated from the use of "@<path>/CBLOPT", instead putting those values here.

Reply:
To be honest, the only time I have played with option files is in the tests for the option files.. I generally don't use them. I could look at changing it, but I think its simpler when all the options are in one spot.. Lets discuss this one.

Reply:
Are you OK if I leave this as is or do you want me to change to use the option files?

Reply:
Okay to leave as is.